### PR TITLE
Feature/volunteer create

### PR DESCRIPTION
**Pull request by YiFeiZhang2**
Make create route for volunteer

### CODE REVIEW

File: controllers/volunteer.controller.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    } else {
+        return res.status(400).json({
+            message: "Issue with volunteer creation",
+            data: {}
+        });
+    }
+}
+
+module.exports = {
+    defaultReturn: function (req, res) {
```
Comment (pierreTklein) [inline controllers/volunteer.controller.js]: What's the purpose of `defaultReturn`? I see this in a lot of your controllers.

File: services/volunteer.service.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"use strict";
+const Volunteer = require("../models/volunteer.model");
+
+/**
+ * @function createVolunteer
+ * @param {JSON} volunteerDetails
```
Comment (pierreTklein) [inline services/volunteer.service.js]: What's expected of volunteerDetails?

File: controllers/volunteer.controller.js (hunk 1 above)
Comment (YiFeiZhang2) [reply controllers/volunteer.controller.js]: I'm really not sure. I remember suddenly seeing it in the exports of one of my controllers, and I thought there was a specific reason, so I started putting it in all the exports. I'll get rid of them.

File: services/volunteer.service.js (hunk 2 above)
Comment (YiFeiZhang2) [reply services/volunteer.service.js]: Currently, all services follow this format. I will update them all in the future.

Comment (pierreTklein) [reply services/volunteer.service.js]: When are you planning on updating them all?

Comment (YiFeiZhang2) [reply services/volunteer.service.js]: It's top priority.

**Pull request merged**